examples/How_to_use_moderation.ipynb

```diff
@@ -0,0 +1,455 @@
+    "- **User Experience Enhancement:** Improve the user experience in chatbots and automated services by providing responses that are polite, relevant, and free from any unsuitable language or content.\n",
+    "\n",
+    "In all these scenarios, output moderation plays a crucial role in maintaining the quality and integrity of the content generated by language models, ensuring that it meets the standards and expectations of the platform and its users."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Setting moderation thresholds\n",
+    "Our output moderation will assess the LLM's response and block anything scoring a 0.4 or higher in any category. Setting this threshold is a common area for optimization - we recommend building an evaluation set and grading the results using a confusion matrix to set the right tolerance for your moderation. The trade-off here is generally:\n",
+    "\n",
+    "- More false positives leads to a fractured user experience, where customers get annoyed and the assistant seems less helpful.\n",
+    "- More false negatives can cause lasting harm to your business, as people get the assistant to answer inappropriate questions, or provide inappropriate responses.\n",
+    "\n",
+    "For example, on a platform dedicated to creative writing, the moderation threshold for certain sensitive topics might be set higher to allow for greater creative freedom while still providing a safety net to catch content that is clearly beyond the bounds of acceptable expression. The trade-off is that some content that might be considered inappropriate in other contexts is allowed, but this is deemed acceptable given the platform's purpose and audience expectations.\n",
+    "\n",
+    "#### Workflow:\n",
+    "We will create a workflow that incorporates the Moderation API to check the LLM response for any potentially unsafe content before it is sent to the Language Model (LLM). This ensures that only appropriate content is displayed to the user.\n",
+    "\n",
+    "1. Receive an input from the user\n",
+    "2. Send prompt to LLM and generate a response\n",
+    "3. Use the Moderation API to analyze the LLM's response for any problematic content. \n",
+    "3. **Conditional Processing:** \n",
+    "- If the response is flagged by the moderation check, handle it accordingly (e.g., reject the response, show a placeholder message, etc.).\n",
+    "- If the response is not flagged, display it to the user.\n",
+    "\n",
+    "We will demonstrate this workflow with two example prompts:\n",
+    "\n",
+    "- **Bad Prompt:** An example input that should trigger the moderation categories and be blocked from reaching the user.\n",
+    "- **Good Prompt:** An example input that should pass the moderation check and be sent back to the user as a valid response."
```

Comment:
I'd call these "outputs" instead of prompts. (also (nit) ditto for the explanation)